lattices/src/lib.rs

```diff
@@ -28,6 +28,7 @@ mod with_top;
 
 pub use conflict::Conflict;
 pub use dom_pair::DomPair;
+pub use lattices_macro::*;
```

Comment:
Generally, wildcard imports are somewhat frowned upon in many other languages. I'm curious to know what the Rust community thinks of this.

https://www.reddit.com/r/rust/comments/87n2ix/what_is_the_motivation_behind_module_wide_use/

Reply:
Yeah I agree that in general `*` imports are bad, but in this specific case the `lattices_macro` crate only exists because proc macros have to live in a separate crate, and everything in there should be re-exported into `lattice`